Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk/WebKit/CMBaseObjectSPI.h

```diff
@@ -47,6 +47,12 @@ enum {
     kCMBaseObjectError_ValueNotAvailable = -12783,
 };
 
+#if WEBRTC_WEBKIT_BUILD
```

Comment:
It is ok to not have WEBRTC_WEBKIT_BUILD here, `Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk` is not synchronised.
That will remove the need for the xcconfig changes hopefully.
Otherwise, the way to go is to include LibWebRTCMacros.h before using that header.